openlcs/libs/corgi.py

```diff
@@ -161,23 +162,17 @@ def get_sync_fields(self, component_uuid):
             "license_concluded",
```

Comment:
`sync_fileds` is a constant, can we put it at the beginning of the file?

Reply:
yeah could do.